frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -420,7 +416,6 @@ class SegmentsView extends React.Component<Props, State> {
     groupToDelete: null,
     groupsSegmentsVisibilityStateMap: {},
     activeStatisticsModalGroupId: null,
-    expandedGroupKeys: [],
```

Comment:
I made this an memoized function of the component. 

While working with this component I saw the following potential danger: The expanded groups were correctly derived from the props / store state, but having this as local state as well as in the store might lead to a state where the local state might be ahead of the store because some codeline accidentally uses `setState` instead of updating the store and from their receiving an updated state via the `getDerivedStateFromProps` method. 

Moreover, I thought that's how it was done in the Trees tab as well, but seems that I was wrong here. => Feel free to argue about this change and I'll change it back.